students/aravichander/lesson07/Mailroom_unittest.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from Mailroom_lesson07 import createlist,getlist,sumdonor
+
+class MyFuncTestCase(unittest.TestCase):
+	
+	def test_createlist(self):
+		createlist()
+		output = getlist()
+		self.assertEqual(output["Bill Gates"],[10000,12000]) 
+		self.assertTrue(output["Bill Gates"] == [10000,12000]) 
+		self.assertFalse(output["Bill Gates"] == [10000,12001])
```

Comment:
you don't need to test non-equality -- if the result is correct, then it's correct. So the assertEqual is all you need here.